src/optimizer/query_rewrite.c

```diff
@@ -1360,6 +1360,12 @@ qo_reduce_equality_terms (PARSER_CONTEXT * parser, PT_NODE * node, PT_NODE ** wh
   PT_NODE *save_where_next;
   bool copy_arg2;
   PT_NODE *dt1, *dt2;
+  bool cut_off;
+  PT_NODE *expr_prev = NULL;
+  PT_NODE *opd1, *opd2;
+  DB_VALUE *dbv1, *dbv2, dbval_res;
+  PT_MISC_TYPE qualifier = (PT_MISC_TYPE) 0;
+  TP_DOMAIN *domain;
```

Comment:
There are variables that can be removed.
```suggestion
  DB_VALUE *dbv1, *dbv2;
```